src/util/system.cpp

```diff
@@ -318,7 +318,7 @@ const fs::path GetExportDir()
     fs::path path;
     LOCK(cs_args);
     if (mapArgs.count("-exportdir")) {
-        path = fs::system_complete(mapArgs["-exportdir"]);
+        path = fs::weakly_canonical(fs::system_complete(mapArgs["-exportdir"]));
```

Comment:
Can you also make the same change for `ZC_GetParamsDir`, and also check other path-related arguments that might appear in `mapArgs`?

Reply:
I made the change for `-paramsdir`, and couldn't find any other relevant paths.